package.json

```
@@ -1,32 +1,44 @@
 {
   "name": "pastebin",
-  "version": "1.0.0",
-  "description": "Copy images, text, and anything else you want to another device seamlessly and easily",
+  "version": "1.0.0b",
+  "description": "Copy images, text, and entire directories to another device seamlessly and easily",
   "main": "index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "lint": "npx eslint --fix .",
+    "format": "npx prettier --write ."
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Quantalabs/pastebin.git"
+    "url": "git+https://github.com/quantum9innovation/pastebin.git"
```

```suggestion
    "url": "git+https://github.com/Quantalabs/pastebin.git"
```
No, this is my project.